algosdk/abi/contract.py

```diff
@@ -63,6 +63,26 @@ def undictify(d: dict) -> "Contract":
             name=name, desc=desc, networks=networks, methods=method_list
         )
 
+    def get_method_by_name(self, name: str) -> Method:
```

Comment:
Just linking to https://github.com/algorand/algorand-sdk-testing/pull/185/files# there is nothing to do here which has a resolved convo explaining why this interface is "good nuf"